src/main/native/jgskit.mak

```diff
@@ -184,7 +184,9 @@ ${HOSTOUT}/%.o: %.c
 	${CC} ${CFLAGS} ${DEBUG_FLAGS} -c -I${GSKIT_HOME}/inc -I${JAVA_HOME}/include -I${JAVA_HOME}/include/${OSINCLUDEDIR} $< -o $@
 
 javah: dircreate
-	${JAVA_HOME}/bin/javac --add-exports java.base/jdk.internal.misc=ALL-UNNAMED -cp ${JCE_CLASSPATH} \
+	${JAVA_HOME}/bin/javac \
+	--add-exports java.base/sun.security.util=openjceplus \
```

Comment:
This shouldn't be necessary when building with Semeru (which already exports that package to openjceplus), but it won't be sufficient for other JDKs (which don't export that package).

Reply:
Hi Keith. We try to keep this compiling and running on both JDKs with and without the export statements that are built into Semeru. When we test this code on 17 we typically pick a really old Semeru SDK to run testing with just so we can attempt to catch export issues that might show up ( Semeru prior to bundling or exports present ). I also compile this code on Mac which does not bundle OpenJCEPlus or has the exports. I noticed these lines at least in the mac makefile which prompted me to make this change. 

There a bunch of additional exports used for compiling other pieces of code that more closely resemble the Semeru exports in the [pom.xml](https://github.com/IBM/OpenJCEPlus/blob/main/pom.xml) file. 

In this particular spot we are only creating the header for the JNI library so there are limited references needed to other packages just for this part of the JNI code.

It might make sense someday for us to modify the github actions in place to compile and test what we can using a non Semeru JDK in parallel to using Semeru.

Reply:
My main point is that this won't _run_ on non-Semeru JDKs (with command-line options to export that package).

Reply:
Hello, I gave this a try on Temurin hotspot as the SDK for the mvn build and test process using the github action process of this project, and it seemed to build/test ok with the various exports we have included the mvn pom and and in this particular spot. 

Ideally we should build and test like this all the time to ensure this remains the case. Perhaps i am missing the path here that is not expected to work.

Reply:
When using a Semeru runtime, `--add-exports=java.base/sun.security.util=openjceplus` is needed for _**neither**_ `javac` nor `java`; with other runtimes that option is needed for _**both**_. You may have that covered in `pom.xml`, etc, but that doesn't help downstream applications that won't work on non-Semeru runtimes unless the application runs with `--add-exports=java.base/sun.security.util=openjceplus`.

Reply:
Yes agreed other SDKs will need to do a wide array of exports equivalent to what is in the `pom.xml`, and in this location, in order to compile / execute given that they do not have the built in exports that Semeru does.